From d729237e78d636f26041c5888f5248b79127c4b5 Mon Sep 17 00:00:00 2001
From: Paul Barker <paul@paulbarker.me.uk>
Date: Fri, 25 Apr 2014 19:06:04 +0000
Subject: [PATCH] Leave CFLAGS alone

configure.ac shouldn't assume the float ABI on arm will always be softfp.

Signed-off-by: Paul Barker <paul@paulbarker.me.uk>
---
 configure.ac | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5f1ff70..004ccc3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -87,14 +87,6 @@ AS_IF([test "$have_vfp" = "yes"],[fpu="-mfpu=vfp"],
       [])
 
 AC_MSG_NOTICE([host is "${host}"])
-case "${host}" in
-	arm* )
-		CFLAGS="$CFLAGS -mfloat-abi=${float_abi} ${fpu} -std=c99"
-		CCASFLAGS="$CCASFLAGS -mfloat-abi=${float_abi} ${fpu}"
-		;;
-	*)
-		;;
-esac
 
 #if test "$ord_sr" = "no"; then
 #	AC_DEFINE(SFFT_ORD_SR,0,[Define to enable ordinary split radix.])
-- 
1.9.2

